Formula/vim.rb

```diff
@@ -2,8 +2,8 @@ class Vim < Formula
   desc "Vi 'workalike' with many additional features"
   homepage "https://www.vim.org/"
   # vim should only be updated every 50 releases on multiples of 50
```

Comment:
Sorry, just one more thing:

```suggestion
  # vim should only be updated every 25 releases on multiples of 25
```

Reply:
No problem; I should have caught that.

PTAL